src/main/java/se/sundsvall/comfactfacade/integration/comfact/configuration/ComfactErrorDecoder.java

```diff
@@ -0,0 +1,42 @@
+package se.sundsvall.comfactfacade.integration.comfact.configuration;
+
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.soap.MessageFactory;
+
+import org.zalando.problem.Problem;
+import org.zalando.problem.Status;
+
+import comfact.GetSigningInstanceInfoResponse;
+import feign.Response;
+import feign.codec.ErrorDecoder;
+
+public class ComfactErrorDecoder implements ErrorDecoder {
+
+	private final ErrorDecoder defaultDecoder = new Default();
+
+	@Override
+	public Exception decode(final String methodKey, final Response response) {
+		try {
+			final var responseStream = response.body().asInputStream();
+			final var soapMessage = MessageFactory.newInstance().createMessage(null, responseStream);
+
+			final var jaxbContext = JAXBContext.newInstance(GetSigningInstanceInfoResponse.class);
```

Comment:
Make jaxbContext static as it is thread safe and available to reuse to avoid and cost of creating every time (https://stackoverflow.com/questions/7400422/jaxb-creating-context-and-marshallers-cost). Unmarshaller is not threadsafe however and must be created each time.